### PR TITLE
Update to disble blank issue.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: .github/actions/testing_report_generation/app.rb [hunk 1]
```diff
@@ -135,5 +135,5 @@ def get_workflows(client, repo_name)
   client.add_comment(REPO_NAME_WITH_OWNER, last_issue.number,failure_report.get_report)
 # Create a new issue if there exists failed workflows.
 else
```
Comment (paulb777) [inline .github/actions/testing_report_generation/app.rb:137]: Shouldn't this else clause be deleted?  Won't the `create_issue` at line 127 cover all cases where a new issue needs t be created?

**Comment by paulb777** [comment]
@granluo See the question above

**Review by granluo — COMMENTED** [review]
line 127 is under a situation that both failure and success report are nil, and an issue or a comment will be created for no workflows running. The content of the issue is NO_WORKFLOW_RUNNING_INFO in line 127.
The else clause is to create an issue. The content is failure_report.get_report, but we need to make sure is not nil ahead, so the unless there is a guard clause to ensure ``failure_report` is not nil before creating a issue.

File: .github/actions/testing_report_generation/app.rb (hunk 1 above)
Comment (granluo) [reply .github/actions/testing_report_generation/app.rb:137]: > @granluo See the question above

The comment was pending and has been updated.

Comment (paulb777) [reply .github/actions/testing_report_generation/app.rb:137]: Thanks for the clarification.

**Review by paulb777 — APPROVED** [review]

**Comment by granluo** [comment]
Will close the issue #7226 and see if it works tonight.

**Pull request merged**